confluence-syntax-xhtml/src/main/java/org/xwiki/contrib/confluence/parser/xhtml/internal/wikimodel/AttachmentTagHandler.java

```diff
@@ -81,6 +82,14 @@ protected void begin(TagContext context)
 
         if (filenameParameter != null) {
             attachment.filename = filenameParameter.getValue();
+            
+            // set attachment filename as macro parameter if the attachment tag is part of a macro
+            Object macro_obj = context.getTagStack().getStackParameter(CONFLUENCE_CONTAINER);
```

Comment:
Minor codesytle/consistency: we tend to use more `macroObj` than `macro_obj` in variable names.

Reply:
Unsure why I did not camelCase that in the first place, but fixed now.